samples/client/petstore/swift3/default/PetstoreClient/Classes/Swaggers/Models.swift

```diff
@@ -185,6 +202,8 @@ class Decoders {
         // Decoder for ApiResponse
         Decoders.addDecoder(clazz: ApiResponse.self) { (source: AnyObject) -> ApiResponse in
             let sourceDictionary = source as! [AnyHashable: Any]
+
```

Comment:
Going to fix that, it's always the tradeoff between moustache and output file readability ;)